cinema/views.py

```diff
@@ -35,7 +35,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
why you commented this line?

Reply:
I set in the project globally through JSON Web Token (JWT) authentication settings. So I don't need to use TokenAuthentication in each views anymore. I commented on it.

Reply:
@mtiunov  in this case, this line is simply better to delete